components/webdriver_server/actions.rs

```diff
@@ -24,11 +25,91 @@ use crate::{Handler, WebElement, wait_for_script_response};
 static POINTERMOVE_INTERVAL: u64 = 17;
 static WHEELSCROLL_INTERVAL: u64 = 17;
 
+// A single action, corresponding to an `action object` in the spec.
+pub(crate) enum ActionItem {
```

Comment:
In the spec I see both terms "action object" and "action item", and they seem to have different semantics. Could you clarify?